### ISSUE TITLE
Pseudo label problems on self-training of Waymo -> KITTI task

### ISSUE THREAD

**Issue by KevinDuBl** [CVMI-Lab/ST3D#42]
When I got the the model of PVRCNN on Waymo dataset with ROS using the command "sh scripts/dist_train.sh ${GPU_NUMS} --cfg_file cfgs/da-waymo-kitti_models/pvrcnn/pvrcnn_old_anchor_ ros.yaml  --batch_size ${batch_size}" and choose the best model by the command "sh scripts/dist_test.sh ${GPU_NUMS} --cfg_file cfgs/da-waymo-kitti_models/pvrcnn/pvrcnn_old_anchor_ros.yaml --batch_size ${batch_size} --eval_all --set DATA_CONFIG_TAR.FOV_POINTS_ONLY True", I start the self-training by the command "sh scripts/dist_train.sh ${GPU_NUMS}  --cfg_file cfgs/da-waymo-kitti_models/pvrcnn_st3d/pvrcnn_st3d.yaml --batch_size ${batch_size} --pretrained_model ${model_path}". Then I get the error on pseudo label loading:
![image](https://user-images.githubusercontent.com/20028640/168627213-42d0af55-0ce4-4fd8-9335-0c54f2a35a07.png)
It seems that information about the 'pos_ps_box' and 'ign_ps_box' was not saved! Did I do something wrong? How can I fix the problem?
